.golangci.yml

```diff
@@ -0,0 +1,26 @@
+run:
+  timeout: 2m
+
+linters:
+  disable-all: true
```

Comment:
I don't like disabling `errcheck` (and others) in golangci-lint.
I would very much prefer for _internal/registry_ to contain unchanged helm code (if we need changes, we overload by composition in _hypper/pkg_).

Once we have _internal/registry_ with unchanged helm code, we can just ignore that folder with `run.skip-dirs` (see https://golangci-lint.run/usage/configuration/#config-file).

Reply:
@viccuad I like your idea of using skip-dirs on imported 3rd party code.